schema/schema.go

```diff
@@ -0,0 +1,20 @@
+package schema
+
+import (
+	"github.com/ONSdigital/go-ns/avro"
+)
+
+var imageUploadedEvent = `{
+  "type": "record",
+  "name": "image-uploaded",
+  "fields": [
+    {"name": "image_id", "type": "string", "default": ""},
+    {"name": "path", "type": "string", "default": ""},
+	{"name": "filename", "type": "string", "default": ""}
```

Comment:
we will need to add this line to `dp-image-api` schema (at the moment it only contains `image_id` and `path`)

Reply:
Thanks for spotting this, it's actually in a related PR (https://github.com/ONSdigital/dp-image-api/pull/42) which I intend to release at the same time as this one. Should have probably linked it in the description.